packages/api/src/routes/blog-subscribe.js

```diff
@@ -0,0 +1,86 @@
+import { JSONResponse } from '../utils/json-response.js'
+import btoa from 'btoa'
+
+const SERVER_PREFIX = 'us5'
+const LIST_ID = '64f6e3fd11'
+const API_KEY = process.env.MAILCHIMP_API_KEY
+const TOKEN = btoa(`:${API_KEY}`)
+
+const headers = {
+  'Content-Type': 'application/json; charset=UTF-8',
+  Authorization: `Basic ${TOKEN}`,
+}
+
+/**
+ *  @param {string} email
+ */
+export const getMailChimpUser = async (email) => {
```

Comment:
This should return a user object or undefined.

Reply:
refactored to just check if the user exists